pretty/src/pretty/pretty_doc.rs

```diff
@@ -107,6 +107,14 @@ pub trait PrettyDocument: Sized + Clone {
         loc_cursor(&root, &lay, &path)
     }
 
+    /// Find the minimum height required to pretty-print the document with the given width.
+    fn required_height(&self, width: Col) -> Row {
+        let root = self.root();
+        let bound = Bound::infinite_scroll(width);
+        let lay = Layouts::compute(&root).fit_bound(bound);
```

Comment:
There at least used to be a `Bounds.fit_width(Col)` method. Is that gone now?

Reply:
There's this, but it doesn't seem all that useful:

https://github.com/justinpombrio/synless/blob/3c6f9494963c48153b691097985137c31e4228dc/pretty/src/layout/boundset.rs#L85